assets/js/admin/admin-scripts.js

```diff
@@ -721,7 +672,63 @@ var give_setting_edit = false;
 					});
 				}
 			}).change();
+		},
+
+		saveButtonTriggered: function() {
+			$( 'body' ).on( 'click', '.give-save-button', function () {
+				$( window ).unbind( 'beforeunload' );
+			});
+		},
+
+		/**
+		 * Show alert when admin try to reload the page with saving the changes.
+		 *
+		 * @since 1.8.14
+		 */
+		changeSettingsUnload: function() {
+			if ( $( 'body' ).closest( '.post-type-give_forms' ).length > 0 ) {
```

Comment:
@mehul0810 Can we improve this selector

Reply:
@ravinderk I have improved the selector. Please check and let me know if there is any further change.